package.json

```diff
@@ -21,7 +21,7 @@
   },
   "private": true,
   "dependencies": {
-    "@educandu/educandu": "~0.0.0-pre.088faa72a80749a378920b29107700edd8eb7e5c",
+    "@educandu/educandu": "~0.0.0-pre.594aae0523ae5a1fe27095dc498ad19519ae8567",
     "@educandu/node-jsx-loader": "~2.1.0",
```

Comment:
Could you please update this to `~2.2.0` to support node >= v16.12.0 ?

Reply:
Sorry, there's more to it, I will make an extra ticket! Please ignore!

Reply:
We can also do this in the next update educandu ticket, we'll need one soon anyway after the next few merges.

Reply:
It's already there: https://educandu.atlassian.net/browse/OMA-5